doc/how_to_introduce_breaking_changes.md

```diff
@@ -29,28 +29,57 @@ You could find the next Breaking Change Release plan in our [milestones](https:/
+All breaking changes **must** be pre-announced **30** days(usually **2** sprints for modules in Core CLI) ahead Release. It gives users the buffer time ahead to mitigate for better command experience. There are two approaches to inform both interactive users and automatic users about the breaking changes.
 
 1. (**Mandatory**) Breaking Changes must be pre-announced through Warning Log while executing.
 2. (*Automatic*) Breaking Changes would be collected automatically and listed in [Upcoming Breaking Change](https://learn.microsoft.com/en-us/cli/azure/upcoming-breaking-changes) Document.
 
+### Breaking Changes in Extensions
+
+All breaking changes in GA extensions **must** be pre-announced at least **30** days prior to their Release. 
+
+Extensions don't need to follow the breaking change window. However, we still strongly recommend releasing breaking changes only in breaking change windows along with Core Azure CLI.
+
+```text
+[GA Release with Breaking Change Pre-Announcement]
+│
+├─ Must include complete Breaking Change Information
+│
+└─┬─ [Minimum 30-day Announcement Period] ────────────────┐
+  │                                                       │
+  │  Allow releases during this period:                   │
+  │  - Other unrelated GA versions (vX.(Y+1), v(X+1).Y)   │
+  │  - Multiple preview releases (Beta)                   │
+  │                                                       │
+  ▼                                                       ▼
+[GA Release Containing Breaking Changes]
+(Must fulfill 30-day announcement requirement)
+```
+
 ## Workflow
 
-### Overview
+### CLI Workflow Overview
 
-* CLI Owned Module
-  * Service Team should create an Issue that requests CLI Team to create the pre-announcement several sprints ahead Breaking Change Window. The issue should include the label `Breaking Change`. The CLI team will look at the issue and evaluate if it will be accepted in the next breaking change release.
+* **CLI Owned Module**
+  * Service Team should create an Issue that requests CLI Team to create the pre-announcement at least **2** sprints ahead of Breaking Change Window. The issue should include the label `Breaking Change`. The CLI team will look at the issue and evaluate if it will be accepted in the next breaking change release.
     * Please ensure sufficient time for CLI Team to finish the pre-announcement.
-  * The pre-announcement should be released ahead of Breaking Change Window.
-* Service Owned Module
-  * Service Team should create a Pull Request that create the pre-announcement several sprints ahead Breaking Change Window.
-  * The pre-announcement should be released ahead of Breaking Change Window.
+  * The pre-announcement should be released at least **2** sprints ahead of Breaking Change Window.
```

Comment:
usually I suppose sprint will vary across team, but monthly basis is often adopted.

If we put as 1-month policy above(Line 32), we might keep it aligned here, rather than 2 sprints, right?